test/test_sorted_containers.jl

```diff
@@ -1,18 +1,24 @@
-import Base.Ordering
-import Base.Forward
-import Base.Reverse
-import DataStructures.eq
+using Base: Ordering, Forward, Reverse, ForwardOrdering,
+            ReverseOrdering
+using DataStructures: IntSemiToken
 import Base.lt
```

Comment:
isn't this called `Base.lessthan` now days?

Reply:
I checked `ordering.jl` in `Base`, and I still see `lt`.  Where did you see `lessthan`?